.github/workflows/ros2.yaml

```
@@ -0,0 +1,20 @@
+---
+name: urdfdom ROS2 CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  build_ros1:
```

I guess it is a typo:
```suggestion
  build_ros2:
```